Project.toml

```diff
@@ -13,5 +13,5 @@ Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 CLIMAParameters = "0.2, 0.3, 0.4"
 DocStringExtensions = "0.8"
 SpecialFunctions = "1, 2"
-Thermodynamics = "0.5, 0.6"
+Thermodynamics = "0.5, 0.6, 0.7"
```

Comment:
Shouldn't it be that now we only support v0.7? This `q_sat  = TD.q_vap_saturation(param_set, ts_neq)` won't work with older versions?

Reply:
Ah, yes, thank you!

Reply:
Right. Lets be conservative and bump to 0.7.

I'm honestly surprised how minor the changes are.

Reply:
Ah, we'll need to do a breaking change since we're dropping previous versions.

Reply:
Hope that's okay!

Reply:
👍